tests/modules/nf-core/purecn/coverage/nextflow.config

```diff
@@ -0,0 +1,8 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: PURECN_COVERAGE {
+        ext.args = "--skip-gc-norm"
```

Comment:
Can you also add a test without this argument? That way we know that works too :)

Reply:
Aaah forgot about that one, my bad! No you can leave it as it is now :)